### DISCUSSION TITLE
Project Admin is not active.

### DISCUSSION THREAD

**Discussion by vaishali-sharma-20** [sanjay-kv/Open-source-Practice#774] [Q&A] [answered]
I sent a pull request as well as a text to assign an issue. but the Project not responded, its been more than 24 hrs. He even didn't assign any of the issues to anyone. 
I request you to take action on it. 
Project name: Travel_Website

**Comment by PRAYAG0908** [comment, marked as answer]
added that repository url in https://github.com/GSSoC24/Contributor/discussions/94 gssoc will flag that project

**Comment by aditi75432** [comment]
Wait for some more time .Ensure that your issue is not some some text changes .  The admin might have a lot of workload. Even if admin does not respond, flag the project

**Comment by nehal1222** [comment]
Mail to the project admin if possible, if it's been more time.

**Comment by codeftsaikat** [comment]
Steps to Follow:
Wait a Bit Longer

Open-source maintainers often manage projects in their free time, and delays are common. Wait another 24–48 hours for a response.
Check Project Activity

Look at recent commits, pull requests, and issues to see if the project is active. If it's inactive, the maintainer might not be available.
Follow Up Politely

Comment on your pull request or issue with a polite follow-up. Example:
"Hi, just following up on this PR and issue. Let me know if there’s anything else I can do to assist."
Contact the Maintainer

If the repository has a contact email or social media link, you can reach out gently to inquire about your contributions.
Consider Moving On

If there’s no response after a reasonable period, focus on other active projects. Your time is valuable!
Reporting or Action
If you believe the project is abandoned or if there's a clear violation of open-source norms, you can:

Report Abandoned Projects: There’s no formal way to "report" unresponsive maintainers, but you can share feedback in the community forums or GitHub discussions.
Fork the Project: If it’s valuable, consider forking the repository and continuing the work independently.
